shopelectro/tests/tests_views.py

```diff
@@ -516,3 +519,32 @@ def test_admin_autocomplete_has_no_model_pages(self):
             f'?term={self.QUOTED_SIGNLE_RESULT_TERM}&pageType=category'
         )
         self.assertTrue(len(json_to_dict(response)) == 1)
+
+
+class Redirects(TestCase):
+
+    fixtures = ['dump.json']
+
+    # will be resurrected at rf#140
+    @unittest.expectedFailure
+    def test_redirect_on_existing_page(self):
+        """DB based redirect from existing url should do, but should not avoid it."""
+        # take some existing `url_from`
+        url_from = '/catalog/categories/category-0/tags/6-v/'
```

Comment:
Let's get a category's url from Model: `Category.objects.first().url`

Reply:
@artemiy312 , here i should take not the first category. But the one, that has at least one tag.

And it'll be too complex to take it. We have fixtures in cvs, so we can completely rely on them in code